test/LANDRegistry.js

```diff
@@ -1098,4 +1097,150 @@ contract('LANDRegistry', accounts => {
       await assertRevert(land.assignNewParcel(0, 3, user, sentByOperator))
     })
   })
+
+  describe('ping', () => {
+    it('should set reactivate authorized', async () => {
+      let isReactiveAuthorized = await land.reactivateAuthorized(user)
+      isReactiveAuthorized.should.be.false
+      await land.setReactivateAuthorized(user, true, sentByCreator)
+      isReactiveAuthorized = await land.reactivateAuthorized(user)
+      isReactiveAuthorized.should.be.true
+    })
+
+    it('should set grace period', async () => {
+      let gracePeriod = await land.gracePeriod()
+      gracePeriod.should.be.bignumber.equal(0)
+      await land.setGracePeriod(1, sentByCreator)
+      gracePeriod = await land.gracePeriod()
+      gracePeriod.should.be.bignumber.equal(1)
+    })
+
+    it('should set deem period', async () => {
+      let deemPeriod = await land.deemPeriod()
+      deemPeriod.should.be.bignumber.equal(0)
+      await land.setDeemPeriod(1, sentByCreator)
+      deemPeriod = await land.deemPeriod()
+      deemPeriod.should.be.bignumber.equal(1)
+    })
+
+    it('should return true if decayed LAND', async () => {
+      const assetId = await land.encodeTokenId(0, 1)
+      await land.setGracePeriod(1, sentByCreator)
+      await land.setDeemPeriod(2, sentByCreator)
+      const decayed = await land.hasDecayed(assetId)
+      decayed.should.be.true
+    })
+
+    it('should return false if no gracePeriod set', async () => {
+      const assetId = await land.encodeTokenId(0, 1)
+      const decayed = await land.hasDecayed(assetId)
```

Comment:
Tip: you can use `1` as `assetId`. The `assetId` for every LAND following the coordinates pattern `(0,x)`  is `x`